images/cross-cloud/sle-hpc/ondemand/profiles.yaml

```diff
@@ -10,10 +9,11 @@ profiles:
   Azure:
     description: Azure configuration
     include:
-      - csp/azure/ondemand/hpc
       - csp/azure/addon/azure-kernel
       - csp/azure/addon/azure-tools
       - csp/azure/addon/waagent-rdma
+      - csp/gce/ondemand
```

Comment:
This probably meant to read `azure` instead of `gce` (in which case it would be superfluous). Should be removed.